### PR TITLE
CAN-426, 427, 430, 431

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Manage Files: Frontend design to delete files in list view
Manage Files: Frontend design to delete file in grid view
Manage Files: Implement validation of 5mb on upload
Manage Files: Implement searching with date and Name

### CODE REVIEW

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 1]
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
   const showCreateFolderModal = () => dispatch(showFolderModal());
   const hideCreateFolderModal = () => dispatch(hideFolderModal());
   const dragBoxShow = () => dispatch(showDrageBox());
-  const dagBoxHide = () => dispatch(hideDrageBox());
+  const dragBoxHide = () => dispatch(hideDrageBox());
   const uploadOptionsHide = () => dispatch(hideUploadOptions());
   const uploadOptionsShow = () => dispatch(showUploadOptions());
   const hideButtonAdd = () => dispatch(hideAddButton());
   const shownAddButton = () => dispatch(showAddButton());
   const StatushideFile = () => dispatch(hideFileStatus());
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please change it to camelcase

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 2]
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
...
+              previewPath: item.thumbUrl,
+            })
+          }
+        >
           <EyeTwoTone /> View File
         </a>
       </Menu.Item>
       <Menu.Item>
-        <a>
+        <a
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please use `Link` on all the anchor links

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 3]
```diff
@@ -99,60 +113,134 @@ const UploadFileUI = ({
...
+            navigator.clipboard.writeText(item.name);
+          }}
+        >
           <CopyTwoTone /> Copy Short Code
         </a>
         ,
       </Menu.Item>
       <Menu.Item>
-        <a>
+        <a
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please use `Link` on anchorlinks

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 4]
```diff
@@ -335,13 +456,39 @@ const UploadFileUI = ({
...
+                    className={styles.After_Upload}
+                    style={fileSizeFlag ? { border: "1px solid red" } : {}}
+                    key={i}
+                  >
+                    <CloseCircleOutlined
+                    // onClick={() =>
+                    //   removeFiles(originNode, file, currFileList,)
+                    // }
+                    />
+                    <img src={file.thumbUrl} height="150px" width="140px" />
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: use Image for images

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 5]
```diff
@@ -361,7 +508,7 @@ const UploadFileUI = ({
                         </div>
                       </div>
                       <div className={styles.dropdown}>
-                        <Dropdown overlay={menu(i)} trigger={["click"]}>
+                        <Dropdown overlay={menu(i, item)} trigger={["click"]}>
                           <a
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please use `Link` on anchorlinks

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 6]
```diff
@@ -376,7 +523,7 @@ const UploadFileUI = ({
               ) : afterUpload && !toggleFileView ? (
                 <Card className={styles.files}>
                   <div className={styles.dropdown_menu}>
-                    <Dropdown overlay={menu_files(i)} trigger={["click"]}>
+                    <Dropdown overlay={menu_files(i, item)} trigger={["click"]}>
                       <a
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please use `Link` on anchorlinks

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx [hunk 7]
```diff
@@ -670,15 +827,15 @@ const UploadFileUI = ({
...
-        title={"Create a Folder"}
+        title={editModal ? "Edit your folder name" : "Create a Folder"}
         visible={visible}
         footer=""
-        onCancel={hideCreateFolderModal}
+        onCancel={() => hideCreateFolderModal()}
         width={400}
         closeIcon={<CloseCircleOutlined />}
       >
         <Form
```
Comment (akuks) [inline src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Please move Form in a separate component

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: Done

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: changed to div

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 3 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: changed to div

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 5 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: done

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 7 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: done

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 4 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: done

File: src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx (hunk 6 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/uploadFiles/uploadFilesUI/index.tsx]: done

**Pull request merged**